packages/tinybased/src/lib/tinybased.ts

```diff
@@ -252,6 +252,28 @@ export class TinyBased<
     );
   }
 
+  /**
+   * Adds sets the rows for a table. This will append new rows and overwrite any existing rows
+   *
+   * @param {TTable} tableName - The name of the table
+   * @param {TBSchema[TTable][]} rows - An array of rows which match the type of the table
+   */
+  bulkAdd<TTable extends OnlyStringKeys<TBSchema>>(
```

Comment:
`bulkUpsert` as the actions this performs with the entities are either (a) insertion if the id doesn't exist or (b) updating the record if the id exists